volmdlr/wires.py

```diff
@@ -1373,15 +1373,16 @@ def area(self):
         return abs(u.cross(v)) / 2
 
 
-
-
 class Circle2D(Contour2D):
     _non_serializable_attributes = ['internal_arcs', 'external_arcs',
                                     'polygon', 'straight_line_contour_polygon',
                                     'primitives', 'basis_primitives']
 
-    def __init__(self, center: volmdlr.Point2D, radius: float, name: str = ''):
-        self.center = center
+    def __init__(self, center, radius: float, name: str = ''):
+        if isinstance(center, list):
```

Comment:
Same as previous comment on sequence check : 

```suggestion
        if dc.is_sequence(center):
```